back/package.yaml

```diff
@@ -197,3 +197,17 @@ tests:
     - temporary
     - webdriver
     - yaml
+
+benchmarks:
+  benchmarks:
+    main: Main.hs
+    source-dirs: benchmarks
+    dependencies:
+      - base <5
+      - guide
+      - gauge
+      - hasql
+      - hasql-transaction
+
+    ghc-options:
+    - -O
```

Comment:
I think this won't have any effect if the library is not compiled with `-O`, so you can just remove it and add a short instruction on how to run benchmarks properly to the `README`.